gdrivefs/gdfs/gdfuse.py

```diff
@@ -852,6 +852,10 @@ def mount(auth_storage_filepath, mountpoint, debug=None, nothreads=None,
             else:
                 v = True
 
+            # ignore the "user" option so we can put in fstab easily
+            if k == "user":
```

Comment:
In a previous version I received an error when I put 'user' in the options in fstab, when I wanted to make mounting the GDriveFS user-mountable. I haven't used GDriveFS in a while, so I have no problem if you simply reject it.